pytest.ini

```diff
@@ -5,7 +5,8 @@ python_paths = demo
 ; point at demo app Django settings
 DJANGO_SETTINGS_MODULE = demo.settings.dev
 
-; addopts = -n 8
+; Set up DB by inspecting models rather than running migrations
+addopts = --nomigrations
```

Comment:
Is this so we can run tests on a new model without creating the migration first?

Reply:
Whoops - good point, I should create the migrations